Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,30 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            Person();
+            Person();
+        }
+
+        static void Person()
+        {
+            Console.WriteLine("Enter name");
```

Comment:
I call such code a blob. Try to split code into logical sections, using an empty line. For example: releated to reading name, related to reading age, printing, etc.